bin/reth/src/commands/dump_genesis.rs

```diff
@@ -0,0 +1,57 @@
+//! Command that dumps genesis block JSON configuration to stdout
+
+use crate::args::utils;
+use clap::Parser;
+use reth_primitives::ChainSpec;
+use std::sync::Arc;
+
+/// Dumps genesis block JSON configuration to stdout, test(todo!(abner))
+#[derive(Debug, Parser)]
+pub struct DumpGenesisCommand {
+    /// Görli network: pre-configured proof-of-authority test network
+    #[arg(long, verbatim_doc_comment)]
+    goerli: bool,
+
+    /// Ethereum mainnet
+    #[arg(long, verbatim_doc_comment)]
+    mainnet: bool,
+
+    /// Holesky network: pre-configured proof-of-stake test network
+    #[arg(long, verbatim_doc_comment)]
+    holesky: bool,
+
+    /// Sepolia network: pre-configured proof-of-work test network
+    #[arg(long, verbatim_doc_comment)]
+    sepolia: bool,
```

Comment:
please make `chain` compatible with this, i.e. remove these flags so you can just do `--chain mainnet`

Reply:
I'm happy to make these changes. @mattsse WDYT

Reply:
scratch that, this just complicates things